### ISSUE TITLE
connectionTimeout

### ISSUE THREAD

**Issue by ravisaluru** [mtakaki/dropwizard-hikaricp#8]
I have been looking at the various permutations to set the connectionTimeout value for the hikariConfig. There is no way in the present code it picks the value from the properties. The value always defaults to 30000.

**Comment by mtakaki** [comment]
You're right, @ravisaluru! It looks like I missed this value. I've created the PR #9 and bumping some dependencies too.

**Comment by mtakaki** [comment]
This is fixed on 2.0.1 and I'll port this fix into the other releases too.

**Comment by mtakaki** [comment]
This is also fixed on 1.3.18. I'm closing this ticket, but please feel free to reopen if you are still seeing some issue.

**Comment by ravisaluru** [comment]
I would like to point out, basically, all the parameters which are mentioned as properties in the dropwizard YAML config aren't propagated as well. maybe look into the properties of the list Properties and set the related values? All these settings aren't really propagated to the Hikari datasource.

**Comment by mtakaki** [comment]
Oh really? In this case it's a big deal and it should be fixed. I'll re-open this issue then.

**Comment by ravisaluru** [comment]
Can I please push the change, I know the exact code we need to change to make the changes.

All we need to do is change the [code here](https://github.com/mtakaki/dropwizard-hikaricp/blob/96a0935358764075fd198310e7fbd5d3dc93d210/src/main/java/io/dropwizard/db/DataSourceFactory.java#L533), 
 from
```java
final HikariConfig config = new HikariConfig();
config.setDataSourceProperties(properties);
```
to 
```java
final HikariConfig config = new HikariConfig(properties);
```

By doing this, the properties which are being passed during the config are correctly set.
